presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java

```diff
@@ -226,7 +226,9 @@ private static Session createSession(Optional<SelectedRole> role)
     {
         return testSessionBuilder()
                 .setIdentity(Identity.forUser("hive")
-                        .withRoles(role.map(selectedRole -> ImmutableMap.of("hive", selectedRole))
+                        .withRoles(role.map(selectedRole -> ImmutableMap.of(
+                                HIVE_CATALOG, selectedRole,
+                                HIVE_BUCKETED_CATALOG, selectedRole))
```

Comment:
Is the bucketed catalog guaranteed to always exist?

Maybe, instead, apply the necessary setup within the test where you need this?

Reply:
It is defined here. So I think it should exist always. I can move this to test, but I do not see feel it is better. The proposed change seem like a decent default to me.